marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs

```diff
@@ -18,11 +18,18 @@
 declareUpdate();
 
 const ds = require("/data-hub/5/data-services/ds-utils.sjs");
+const Security = require("/data-hub/5/impl/security.sjs");
 
 var saveQuery;
 var userCollections = ["http://marklogic.com/data-hub/saved-query"];
 var queryDocument = JSON.parse(saveQuery);
 
+let authorized = new Security().getRolesAndAuthorities().authorities.includes("canManageSavedQuery");
```

Comment:
One small note, and no need to change (I'm going to approve) - for variables that aren't going to change, prefer "const" over "let". It makes for more readable code too, as I can see this and know it's never going to change.